modules/sitemaps/sitemap-constants.php

```diff
@@ -7,6 +7,16 @@
  * @author Automattic
  */
 
+/**
+ * Number of seconds between sitemap and news updates in development code.
+ *
+ * @since 5.x.x
```

Comment:
This would be 7.7.0:

```suggestion
 * @since 7.7.0
```